src/components/Person/Person.jsx

```diff
@@ -1 +1,15 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    {person.age && (
+      <p className="Person__age">{`I am ${person.age}`}</p>
+    )}
+    <p className="Person__partner">
+      {person.isMarried ? (
+        `${person.partnerName} is my ${person.sex === 'f' ? ('husband') : ('wife')}`
```

Comment:
Create a constant for a partner outside the JSX and move this logic there. The JSX code should be clean and focused on rendering components rather than defining complex logic directly within it.

Reply:
+